### DISCUSSION TITLE
is it possible to access HFS the same as cloud storage or WebDav?

### DISCUSSION THREAD

**Discussion by mitobin** [rejetto/hfs#417] [Q&A]
is it possible to access HFS the same as cloud storage or WebDav?

**Comment by rejetto** [comment]
hey, webdav is not supported yet. It's very possible that it will be in the future, but it won't happen soon.
I don't know what you mean with cloud storage.

**Reply by mitobin** [reply]
I want to sync data from my Android device to HFS via a third app, but HFS doesn't support those protocols. Synchronizing and sharing data on mobile devices is very popular, it would be great if HFS could do that.

**Comment by rejetto** [comment]
It could be useful to know what the other protocol is. Cloud storage is not a protocol.

**Comment by mitobin** [comment]
for example this picture
![IMG_20231225_211043](https://github.com/rejetto/hfs/assets/132363335/80d5fde4-7b77-4d07-8d98-9d499cae7583)

**Comment by mitobin** [comment]
The app supports only ftp, webdav protocols and some types of cloud storage accounts. Unable to connect to HFS to backup and synchronize data

**Reply by rejetto** [reply]
i see

**Comment by mitobin** [comment]
Hi Rejetto. Is there any solution to this problem?

**Comment by rejetto** [comment]
not really.
i guess you have to wait until someone works on this specific feature (probably me, later in the year).

**Reply by mitobin** [reply]
Thank you for putting this on your to-do list. very excitedly waiting

**Comment by mitobin** [comment]
I find this Webdav server software sharing channel quite simple. I installed it but can't login to webdav server. I wonder if you have any ideas?
https://github.com/hacdias/webdav

**Comment by rejetto** [comment]
i don't know that software and never used it, so i have no idea

**Comment by mitobin** [comment]
Oh. I just put it up for reference, don't know if it's useful. Sorry if it bothers you

**Reply by HiCyprus** [reply]
Try google search: WebDAV Chfs. CHFS serves as webDav alright, but much less powerful than HFS. Hoping HFS's Webdav function asap.

**Reply by mitobin** [reply]
Thanks. but I don't understand the command, so I can't use it.

**Reply by HiCyprus** [reply]
no need to remember the command. if you are using windows, CHFS has GUI very user friendly. Try Google seearch: webdav  chfs. Considering safety, free SSL certificate recommended.

**Reply by HiCyprus** [reply]
http://iscute.cn/chfs； choose GUI-CHFS-WINDOWS.ZIP. In this GUI, can choose English language.

**Reply by mitobin** [reply]
Thanks . it works well and is simple. but Webdav only displays the parent folder, subfolders are not displayed with the Webdav protocol. and I can't find a place to change the https port.

**Reply by HiCyprus** [reply]
It's great to have CHFS as a supplementary webDAV server running together with HFS. The creator of CHFS seems to have abandoned this project and am really sorry we have only primative vertion 1.0, with problems in adding directaries to the virtual file system. Anyway, it serves well as webDAV for our HFS. Will send you email discussing more detail about settings of CHFS.

**Reply by mitobin** [reply]
I created an account but I can still access files without using the web account. it has to do with security and there are a lot of things missing. Thank you for your enthusiastic help

**Reply by HiCyprus** [reply]
I wrote an email and sent it to mitobin (notifications@github.com). But this email address seems not yours. please check. if you don't receive it, I will try resend it to you or simply use Hfs link.

**Reply by mitobin** [reply]
Yes. That's not my email address. Can you please send me the instructions via the HFS link? Thanks a lot!

**Reply by HiCyprus** [reply]
okay! https://public.xies.uk/mail.zip
user: xuser; password: abcd1234

**Reply by mitobin** [reply]
Thank. I understood what you instructed. But I have problem accessing http protocol. I have created an account and granted permissions to access the shared folder, but when accessing via http, I can still access the folder without needing to log in. webdav has no problem.
Second, when changing the port for http and https, there will be 2 places to change the port (eg http:80 and https:443) but in the software there is only 1 place to change. i don't know if it is a change of http or https.
Best regards!

**Reply by HiCyprus** [reply]
As I wrote in the email, for the first problem, you should change the permission for VISITORS to "Not Readable" first; for the second problem, as system default, HTTP uses port 80; while HTTPS uses port 443. As I have already recommended to you in the mail, as safety is concerned, HTTPS is strongly rwcommended. and the default port of 443 be changed. Anyway, you can copy the setting of your CHFS, and write in detail the problems you meet, and upload the doc to my HFS website. I will see to it should I have time.

**Reply by mitobin** [reply]
Thank you for your help. All problems have been resolved. Wishing you health and happiness

**Reply by HiCyprus** [reply]
Hope HFS webdav coming out quick and better!

**Reply by mitobin** [reply]
I'm also looking forward to it

**Comment by rejetto** [comment]
ok i see, but no, it's not useful

**Comment by HiCyprus** [comment]
CHFS serves as webDav alright, but much less powerful than HFS. I have to run HFS and CHFS together. Hoping HFS's Webdav function asap.
